Carpeta donde estarán los archivos del XBlock y el archivo readme par…

LeandroPotes opened the pull request:
…a el XBlock generado

J4ckDev committed:
Carpeta donde estarán los archivos del XBlock y el archivo readme para el XBlock generado

LeandroPotes committed:
Add files via upload

J4ckDev commented on TEA/adaptive_test/adaptive_test/static/js/src/student_adaptive_test.js:
```diff
@@ -0,0 +1,328 @@
+    }
+
+    function loadHoneyAlonso() {
+        html = '<div id="testContainer"> <div class="card-panel teal light-blue center-align flow-text white-text text-light-blue"> TEST HONEY-ALONSO </div> <p class="">Este cuestionario no ha sido diseñado para identificar tu estilo preferido de aprender. No es un test de iinteligencia, ni de personalidad. </p> <p class="">No hay respuestas correcta o erroneas. Será útil en la medida que seas si ...
```
La pregunta 19 tiene repetido su texto 2 veces, las preguntas 22 y 25 tienen la letra r de sobra.

J4ckDev reviewed (commented):
Solo se debe realizar un cambio menor en la función de Honey-Alonso, por lo demás funcionó correctamente en las pruebas realizadas

J4ckDev reviewed (commented):
Arreglar los detalles de student_adaptive_test.js

J4ckDev reviewed (changes requested):
Corregir lo anteriormente mencionado

LeandroPotes committed:
Update student_adaptive_test.js

Correcciones en el HTML de la función loadHoneyAlonso realizadas.

J4ckDev reviewed (approved):
Cambios realizados correctamente.

LeandroPotes committed:
Add files via upload

J4ckDev commented on TEA/adaptive_test/adaptive_test/adaptive_test.py:
```diff
@@ -0,0 +1,162 @@
+        #Database, user and password must be changed according to the local database
+        conn = psycopg2.connect(database='db_user',user='postgres',password='leandro21020',host='localhost')
+        cur = conn.cursor()
+        cur.execute("INSERT INTO resultadostest (fecha, nombre_test, resultado) VALUES (CURRENT_DATE,%s, %s)", (data['test_name'], data['result']))
```
Agregar el ID para quitar el valor autoincremental en la base de datos

LeandroPotes committed:
Cambios con id_usuario y vista del profesor

Se suben a la base de datos el id_usuario obtenido por el xblock, se puede comparar si cada usuario ha presentado o no los respectivos test para mostrar el resultado o el test en caso contrario.
Se modificó el studio_view añadiendo una vista(studio_analytics) para permitir que el docente mire los resultados de los tests y pueda volver a elegir otro test.
